.github/workflows/html_publisher.yaml

```diff
@@ -0,0 +1,39 @@
+name: HTML Publisher
+on:
+  push:
+    paths:
+    - "notebooks/database-builds.ipynb"
+    - "notebooks/database-schema.ipynb"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    name: Publish HTML notebooks
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
```

Comment:
Any reason to not run this on 3.9?

Reply:
No, none at all. I grabbed this from the, now deleted, default actions that came from the template. I'll update it before I merge.